lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -430,8 +429,8 @@ def _section_course_info(course, access):
         'course_display_name': course.display_name,
         'has_started': course.has_started(),
         'has_ended': course.has_ended(),
-        'start_date': get_default_time_display(course.start),
-        'end_date': get_default_time_display(course.end) or _('No end date set'),
+        'start_date': course.start,
+        'end_date': course.end or None,
```

Comment:
Isn't course.end a datetime object?  In which case it's already going to be None if it evaluates as False, right?